app/src/main/java/com/alfame/ville/memo/Note.java

```diff
@@ -18,17 +18,35 @@
     DateFormat dateFormat = new SimpleDateFormat("dd/MM/yyyy");
 
 
-    Note(String title, String text, String category){
+    //dev phase constructor
+    Note(int id,String title,String text,String category, Boolean struck){
+        this.id=id;
         this.title=title;
         this.text=text;
         this.category=category;
-        this.priority=priority;
-        this.struck= false;
```

Comment:
I changed struck to false because its always false when using the constructor, I don't see the reason to pass it to the constructor in that case. No big deal, no need to fix.